packages/ws/src/ws/WebSocketManager.ts

```diff
@@ -43,6 +43,10 @@ export interface SessionInfo {
 	 * The total number of shards at the time of this shard identifying
 	 */
 	shardCount: number;
+	/**
+	 * URL to use when resuming
+	 */
+	resumeUrl: string;
```

Comment:
For consistency with the rest of the lib
```suggestion
	resumeURL: string;
```